src/win/util.c

```diff
@@ -1135,6 +1136,13 @@ int uv_getrusage(uv_rusage_t *uv_rusage) {
     return uv_translate_sys_error(GetLastError());
   }
 
+  ret = GetProcessMemoryInfo(GetCurrentProcess(),
+                             &memCounters,
+                             sizeof(PROCESS_MEMORY_COUNTERS));
```

Comment:
Sorry, I was just talking about the wrong line before. I amended the change. Undoubtedly right. Thx.